R/pmcmc.R

```diff
@@ -119,20 +119,43 @@ pmcmc <- function(pars, filter, n_steps, save_state = FALSE,
     }
   }
 
-  pars <- set_colnames(list_to_matrix(history_pars$get()),
-                       names(curr_pars))
+  pars_matrix <- set_colnames(list_to_matrix(history_pars$get()),
+                              names(curr_pars))
   probabilities <- set_colnames(list_to_matrix(history_probabilities$get()),
                        c("log_prior", "log_likelihood", "log_posterior"))
 
-  state <- trajectories <- NULL
+  predict <- state <- trajectories <- NULL
```

Comment:
Thoughts about a default of `save_state = TRUE`? If you forget to set it that's going to be annoying and a loss of time, whereas the overhead would be relatively small

Reply:
yeah, that's a good argument. Updated in b8b8e29, along with a comment about the consequences